client/app/components/custom-button/custom-button.directive.js

```diff
@@ -2,10 +2,10 @@
   'use strict';
 
   angular.module('app.components')
-    .directive('customButton', CustomButtonDirective);
+    .directive('customButton', ['$window', '$timeout', CustomButtonDirective]);
```

Comment:
Shouldn't need that array, if it doesn't work without it, it means `@ngInject` is broken..